keras/layers/convolutional.py

```diff
@@ -277,6 +277,9 @@ class Conv1D(_Conv):
             (see [constraints](../constraints.md)).
         bias_constraint: Constraint function applied to the bias vector
             (see [constraints](../constraints.md)).
+        causal: Boolean, whether layer uses causal convolutions.
+            E.g. output[t] depends solely on input[:t-1].
+            Requires padding='valid'.
```

Comment:
Use code formatting in the docstring (`) around arguments/etc. Also, please add a bit more information to the docstring so that someone would doesn't know about causal convolutions would understand vaguely what it's about from reading the docstring.